### ISSUE TITLE
LegacyFaceDetectMapper: onFaceDetection - Ignored some incoming faces sinceface detection was disabled

### ISSUE THREAD

**Issue by aaronhan92** [szaza/android-yolo-v2#14]
Hi szaza, do you have any idea how to enable the onFaceDetection so that my face objection can work

**Comment by szaza** [comment]
What do you mean under _onFaceDetection_?
This project does not contain any face detection feature. Did you extended it?
Can you provide me more details about your work?

**Comment by aaronhan92** [comment]
Goal: To implement face detection and facial expression recognition using Darkflow.
Please kindly comment whether if my implementation flow works as follow:

1) Git clone the darkflow.git

2) Download pre-trained tiny-yolo-widerface.cfg and tiny-yolo-widerface_final.weights from https://drive.google.com/file/d/1wAy6-XWDBHMMawgPfKv2hp_Q5NBYqK1n/view (Able to detect face with bounding box that trained on WiderFace dataset)

3) Prepare my own small dataset (also copy from Widerface) using LabelImg software tool to crop and label each facial expression such as happy and sad etc.

4) Start to train my model using this command 'python flow --model cfg/tiny-yolo-widerface.cfg --train --trainer adam --annotation train_set/Annotation --dataset training_set/Images'.

5) Generate the pb file using this command 'python flow --model cfg/tiny-yolo-widerface.cfg --load 8000 --savepb

6) Place the generated pb file to the android asset folder

**Comment by aaronhan92** [comment]
Yes, I'm able to use the generated model to detect my face via web cam. Moreover, after generated the pb file using this command 'python flow --model cfg/tiny-yolo-widerface.cfg --load 8000 --savepb', I placed it to the asset folder and start to run the android studio. 

However, during debug phase, I encountered this error message 'LegacyFaceDetectMapper: onFaceDetection - Ignored some incoming faces sinceface detection was disabled' (I think the problem exist is due to Camera API 2)
